frontend/src/components/Home.js

```diff
@@ -69,6 +74,12 @@ export class Home extends Component {
         </div>
         {this.state.isIE && <WarningBanner onClose={this.handleIEClose} type="IE" />}
         {this.state.isTest && <WarningBanner type="test" />}
+        <MediaQuery maxWidth={769}>
```

Comment:
update to be 500, or 501. 
769 is tablet width

Reply:
done